app/models/manageiq/consumption/consumption_manager.rb

```diff
@@ -12,9 +13,10 @@ def self.description
   end
 
   def self.update_events
+    data_units = load_column_units
```

Comment:
@lpichler Do you think we should memoize this?
How is it done in other configurations in the application?

Reply:
I didn\`t see it on other places in app for these reasons but I think It can be memoized(I mean memoize method load_column_units). so you can pass it directly  
to `event.update_event(`